app/models/user_role.rb

```diff
@@ -165,6 +166,12 @@ def set_position
     self.position = -1 if everyone?
   end
 
+  def validate_own_role_edition
+    return unless defined?(@current_account) && @current_account.user_role.id == id
+    errors.add(:permissions_as_keys, :dangerous) if permissions_changed?
```

Comment:
I think you meant to use `:own_role` here instead of `:dangerous`

Reply:
Yes indeed, sorry about that!